src/EPR.SubsidiaryBulkUpload.Application/Services/BulkUploadOrchestration.cs

```diff
@@ -24,6 +25,20 @@ public BulkUploadOrchestration(IRecordExtraction recordExtraction, ISubsidiarySe
 
     public async Task NotifyErrors(IEnumerable<CompaniesHouseCompany> data, UserRequestModel userRequestModel)
     {
+        if (data.Count() == 0)
+        {
+            var fileValidaditon = new List<UploadFileErrorModel>();
```

Comment:
Spelling - fileValidaditon  should be fileValidation

Reply:
changed. lol